scripts/gen_dockerfiles.py

```diff
@@ -326,9 +341,10 @@ def build_rpm_final_cmd(features: list, selinux: bool) -> str:
 # Change directory to the repository
 WORKDIR /himmelblau
 
-# Install Rust (latest stable)
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \\
-    cargo install cargo-deb cargo-generate-rpm
+# Install the cargo-deb and cargo-generate-rpm tools
+RUN --mount=type=cache,target=/usr/local/cargo/registry cargo install cargo-deb cargo-generate-rpm
```

Comment:
I'm confused by the syntax here. I thought `RUN` was a shell command? Why here does the command start with parameters? Is this some Docker-ism that I'm just not familiar with?